core/client/app/controllers/settings/labs.js

```diff
@@ -35,6 +35,16 @@ var LabsController = Ember.Controller.extend(Ember.Evented, {
         return this.get('controllers.feature.codeInjectionUI') || false;
     }),
 
+    usePassProtectUI: Ember.computed('controllers.feature.passProtectUI', function (key, value) {
+        // setter
+        if (arguments.length > 1) {
+            this.saveLabs('passProtectUI', value);
```

Comment:
In a setter situation, you want to also return the value so that it can be cached. Just follow this line up with `return value;`